qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -826,6 +826,151 @@ def __new__(
         return instance
 
 
+def GaussianSquareDrag(
+    duration: Union[int, ParameterExpression],
+    amp: Union[float, ParameterExpression],
+    sigma: Union[float, ParameterExpression],
+    beta: Union[float, ParameterExpression],
+    width: Optional[Union[float, ParameterExpression]] = None,
+    angle: Optional[Union[float, ParameterExpression]] = 0.0,
+    risefall_sigma_ratio: Optional[Union[float, ParameterExpression]] = None,
+    name: Optional[str] = None,
+    limit_amplitude: Optional[bool] = None,
+) -> SymbolicPulse:
+    """A square pulse with a Drag shaped rise and fall
+
+    This pulse shape is similar to :class:`~.GaussianSquare` but uses
+    :class:`~.Drag` for its rise and fall instead of :class:`~.Gaussian`. The
+    addition of the DRAG component of the rise and fall is sometimes helpful in
+    suppressing the spectral content of the pulse at frequencies near the
+    fundamental frequency of the drive which can cause unwanted excitation of
```

Comment:
In my understanding DRAG pulse effectively works as a frequency filter and creates a notch at some frequency. This only aims at narrowing the spectral component around the fundamental frequency?

Reply:
I can try to make this a little clearer. I meant the same thing that you said. DRAG can create a notch at a frequency near the fundamental but a little bit off from it. That can help when there is a spectator 20-50 MHz away from the drive frequency, so it is close to the drive but not too close. If it were closer, then DRAG wouldn't be able to keep it from being driven.

Reply:
Thanks. I like new description.